ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -107,7 +107,7 @@ presubmits:
       - command:
         - ci-operator
         - --artifact-dir=$(ARTIFACTS)
-        - --secret-dir=/usr/local/e2e-aws-smoke-cluster-profile
+        - --secret-dir=/usr/local/e2e-aws-cluster-profile
```

Comment:
These have to be different because the names can't collide with e2e-aws - this is for namespacing the keys in the target namespace ci-operator uses.

Reply:
It's template and target that have to be different.  Secrets are fine as long as other jobs that create the secret with this name don't have different contents